pkg/integrations/redis_exporter/redis_exporter.go

```diff
@@ -135,11 +142,15 @@ func New(log log.Logger, c *Config) (integrations.Integration, error) {
 	}
 
 	if c.ScriptPath != "" {
-		ls, err := os.ReadFile(c.ScriptPath)
-		if err != nil {
-			return nil, fmt.Errorf("Error loading script file %s: %w", c.ScriptPath, err)
+		scripts := map[string][]byte{}
+		for _, path := range strings.Split(c.ScriptPath, ",") {
+			ls, err := os.ReadFile(path)
```

Comment:
Are we really going to load every file in a directory? Seems like a potential security issue. Especially if file names and contents get into the logs on parse errors.

Reply:
It shouldn't be loading every file in the directory, just loading every file in a comma-separated list specified in the config. So as an example if provided with the config:
```yaml
  [...]
  script_path: "/tmp/redis-metrics.lua,/tmp/extra-redis-metrics.lua"
```
This would be splitting `"/tmp/redis-metrics.lua,/tmp/extra-redis-metrics.lua"` into `[]string{"/tmp/redis-metrics.lua", "/tmp/extra-redis-metrics.lua"}` and reading each of those.